taxonomy/migrations/0001_initial.py

```diff
@@ -9,7 +9,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('learningresources', '0001_initial'),
+    #    ('learningresources', '0001_initial'),
```

Comment:
Thanks! I had to comment it out temporarily to recreate the `learningobjects` migration and failed to fix it.